bam_stat.c

```diff
@@ -207,11 +237,15 @@ static void out_fmt_json(bam_flagstat_t *s) {
 static void out_fmt_tsv(bam_flagstat_t *s) {
     char b0[16], b1[16];
     printf("%lld\t%lld\ttotal (QC-passed reads + QC-failed reads)\n", s->n_reads[0], s->n_reads[1]);
+    printf("%lld\t%lld\tprimary (QC-passed reads + QC-failed reads)\n", s->n_primary[0], s->n_primary[1]);
```

Comment:
Do we want the "(QC-passed reads + QC-failed reads)" text on this line?  It doesn't appear on the default format.